recipes/pandarallel/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "pandarallel" %}
+{% set version = "1.5.1" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pandarallel-{{ version }}.tar.gz
+  sha256: 28c101e4eeda6a2b49888db413bc16e28bf36feb3e779d58c099ee49cde83849
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.5
+  run:
+    - dill
+    - pandas
+    - python >=3.5
+
+test:
+  imports:
+    - pandarallel
+    - pandarallel.data_types
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/nalepae/pandarallel
+  summary: An easy to use library to speed up computation (by parallelizing on multi CPUs) with pandas.
+  license: BSD-3-Clause
+  license_file: LICENSE
```

Comment:
Add comment above `license_file`:

```
# license file manually packaged. See https://github.com/nalepae/pandarallel/pull/113
license_file: LICENSE
```

I've submitted a PR to include the license file in the source distribution (https://github.com/nalepae/pandarallel/pull/113). The comment will be a reminder to check the status of this on future releases of the package and updates to the feedstock.

Reply:
Thank you for having a look but I tried that lib locally and it hangs forever when using it. I don't have time to debug this so I will just close this PR for the moment.

Reply:
(sorry I should have close it before...)